checks/http/oracle.rb

```diff
@@ -206,10 +206,10 @@ def generate_checks(url)
               description: 'server header',
               references: [],
               match_type: :content_headers,
-              match_content: %r{server: SAP NetWeaver ApplicationServer [\d.]+ / AS Java [\d.]+}i,
+              match_content: %r{server: SAP NetWeaver Application Server [\d.]+ / AS Java [\d.]+}i,
```

Comment:
same here, fingerprint broken?

Reply:
This was broken in a previous commit I did, so I adjusted them back.